Sources/Vapor/Response/ResponseCodable.swift

```diff
@@ -10,7 +10,15 @@ public protocol ResponseEncodable {
     func encodeResponse(for request: Request) -> EventLoopFuture<Response>
 }
 
+/// Can convert `Request` to a `Self`.
+///
+/// Types that conform to this protocol can be returned in route closures.
```

Comment:
This is wrong - it should be 
```suggestion
/// Types that conform to this protocol can decode requests to their type.
```